kinetic/distribution.yaml

```diff
@@ -4541,6 +4541,16 @@ repositories:
       url: https://github.com/yujinrobot/kobuki_soft.git
       version: kinetic
     status: maintained
+  komodo2:
+    doc:
+      type: git
+      url: https://github.com/robotican/komodo_2.0.git
+      version: kinetic
```

Comment:
there is currently no `kinetic` branch on the `robotican/komodo_2.0` repository, did you mean to use `master`? (same in the source section below)